frontend/rolecall/src/app/performance/performance-editor.component.ts

```diff
@@ -503,6 +503,10 @@ export class PerformanceEditor implements OnInit, OnDestroy, AfterViewChecked {
     }
     this.intermissions = newInterms;
     if (!this.initCastsLoaded) {
+      for (let s = 0; s < this.state.step_3.segments.length; s++) {
```

Comment:
Add a comment above the for-loop like `// Sort groups by position.` or whatever makes sense.

Reply:
Done.